packages/builder/src/schemas.ts

```diff
@@ -80,20 +80,29 @@ export const deploySchema = z
          */
         description: z.string().describe('Description of the operation'),
         /**
-         *    Determines whether contract should get priority in displays
+         * Determines whether contract should get priority in displays
          */
         highlight: z.boolean().describe('Determines whether contract should get priority in displays'),
         /**
-         *    Determines whether to deploy the contract using create2
+         * Determines whether to deploy the contract using create2
          */
         create2: z
           .union([z.boolean(), z.string().refine((val) => isAddress(val))])
           .describe(
             'Determines whether to deploy the contract using create2. If an address is specified, the arachnid create2 contract will be deployed/used from this address.'
           ),
         /**
-         *    Contract deployer address.
-         *    Must match the ethereum address format
+         * Determines whether to deploy the contract using create2
+         */
+        ifExists: z
+          .enum(['continue'])
+          .optional()
+          .describe(
+            'Determines whether when trying to deploy the contract using CREATE2 it should continue if the target address already exists. Disabled by default'
```

Comment:
'When deploying a contract with CREATE2, determines the behavior when the target contract is already deployed (ex. due to same bytecode and salt). Set to `continue` to allow the build to continue if the contract is found to have already been deployed. By default, an error is thrown and the action is halted.'